deploy/runtime/ui/Utility Rate - Unused.txt

```diff
@@ -1,7 +1,7 @@
 Utility Rate - Unused
```

Comment:
Is this the information from the urdb that SAM does not currently use? Looks like it may be getting shorter!

Reply:
haha! They removed one and added two, so still getting longer. These particular parameters are used to represent demand billing components that are different for different utilities. Until the parameters have a consistent meaning, it will be difficult to use them in SAM.